linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_retry/tests/pwquality_conf_overriden.fail.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# packages = authselect
+# platform = Oracle Linux 8,Oracle Linux 9,Red Hat Enterprise Linux 8,Red Hat Enterprise Linux 9
+# variables = var_password_pam_retry=3
+
+source common.sh
+
+echo "retry = 3" >  /etc/security/pwquality.conf
+
+for file in ${configuration_files[@]}; do
+	{{{ bash_ensure_pam_module_option('/etc/authselect/custom/testingProfile/$file',
+                                   'password',
+								   'required',
+								   'pam_pwquality.so',
+								   "retry",
+								   "4",
+								   "4") }}}
```

Comment:
This last `4` is not necessary here. It was inherited from the old macro. However, since it is not actually changing anything in the functionality of this test scenario, I am fine to waive it and keep this as an improvement opportunity for another PR.